agents/components/Profile.js

```diff
@@ -105,11 +105,6 @@ export default compose(
     toggleEdit: ({ setEditing }) => () => setEditing(not)
   }),
   connectForm({
-    form: 'profile',
-    initialValues: {
```

Comment:
right, we [pass in `initivalValues`](https://github.com/root-systems/cobuy/pull/120/files#diff-1f06d808fe3c977d06b08e65d4d1e4d6R27) and [`enableReinitialize`](https://github.com/root-systems/cobuy/pull/120/files#diff-54abf595da195cf7b60bd6b1d767ccb4R117). can either cherry-pick from there, merge that pull request as is and rebase on top, or add those lines here.